.github/workflows/latest-release.yml

```diff
@@ -60,13 +61,72 @@ jobs:
           CHANNEL: ${{ env.CHANNEL }}
           POSTFIX: ${{ steps.set_postfix_value.outputs.postfix }}
           OVERRIDE_PACKAGE_VERSION: true
+
+  build-plugin:
+    name: Build Plugin
+    needs: [configure-env-variables]
+    uses: ./.github/workflows/plugin-build.yml
+    with:
+      PACKAGE_VERSION: ${{ needs.configure-env-variables.outputs.PACKAGE_VERSION }}
+      PACKAGE_FILES_VERSION: ${{ needs.configure-env-variables.outputs.PACKAGE_FILES_VERSION }}
+
+  latest_release:
+    if: github.event.pull_request.merged == true && github.base_ref == 'main' && !contains(github.event.pull_request.labels.*.name, 'skip release' )
+    needs: [ build-plugin, configure-env-variables ]
+    env:
+      PACKAGE_VERSION: ${{ needs.configure-env-variables.outputs.PACKAGE_VERSION }}
+      CLEAN_PACKAGE_VERSION: ${{ needs.configure-env-variables.outputs.CLEAN_PACKAGE_VERSION }}
+      CURRENT_CHANNEL_VERSION: ${{ needs.configure-env-variables.outputs.CURRENT_CHANNEL_VERSION }}
+      CLEAN_REF: ${{ needs.configure-env-variables.outputs.CLEAN_REF }}
+      PLUGIN_ZIP_FILENAME: ${{ needs.configure-env-variables.outputs.PLUGIN_ZIP_FILENAME }}
+      PLUGIN_ZIP_PATH: ${{ needs.configure-env-variables.outputs.PLUGIN_ZIP_PATH }}
+      PREVIOUS_TAG_SHA: ${{ needs.configure-env-variables.outputs.PREVIOUS_TAG_SHA }}
+      BETA_VERSION: ${{ needs.configure-env-variables.outputs.BETA_VERSION }}
+    permissions: write-all
+    runs-on: ubuntu-22.04
+    steps:
+      - name: checkout branch
+        uses: actions/checkout@v4
+        with:
+          ref: main
+
+      - name: Get Configuration
+        id: configuration
+        uses: rgarcia-phi/json-to-variables@9835d537368468c4e4de5254dc3efeadda183793
+        with:
+          filename: './.github/workflows/config.json'
+          prefix: config
+
+      - name: Install Node.js 20.x
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.x
+
       - name: Install Dependencies
         run: npm ci
-      - name: Build plugin
-        uses: ./.github/workflows/build-plugin
+
+      - name: Update Beta tag in Readme.txt file
+        run: |
+          if grep -q "Beta tag:" ./readme.txt
+            then
+              echo "Replace existing beta tag in readme file"
+              sed -i -E "s/Beta tag: .*/Beta tag: ${{ env.BETA_VERSION }}/g" ./readme.txt
+            else
+              echo "Add beta tag to readme file"
+              sed -i -E '/^Stable tag: .*/a\'$'\n'"Beta tag: ${{ env.BETA_VERSION }}$(printf '\n\r')" ./readme.txt
+          fi
```

Comment:
Not sure that this should be here
There is a comment regarding this in the other location.